jsonplugin/unmarshal.go

```diff
@@ -156,6 +156,26 @@ func (s *UnmarshalState) ReadFloat32() float32 {
 	}
 }
 
+// ReadWrappedFloat32 reads a wrapped float32 value. This also supports string encoding as well as {"value": ...}.
+func (s *UnmarshalState) ReadWrappedFloat32() float32 {
+	if s.Err() != nil {
+		return 0
+	}
+	if s.inner.WhatIsNext() != jsoniter.ObjectValue {
+		return s.ReadFloat32()
+	}
+	if key := s.ReadObjectField(); key != "value" {
+		s.SetErrorf("first field in wrapped float32 is not value, but %q", key)
+		return 0
+	}
+	v := s.ReadFloat32()
+	if field := s.ReadObjectField(); field != "" {
```

Comment:
Does `jsoniter` support empty object keys, or are they dropped by default ?

Reply:
Not sure... I guess it doesn't allow those, because Godoc says:

```
// ReadObject read one field from object.
// If object ended, returns empty string.
// Otherwise, returns the field name.
```